Dockerfile

```diff
@@ -24,14 +24,18 @@ RUN \
     echo '' > requirements.txt && \
     pip3 install . --no-deps
 
-RUN pip install jsonformatter==0.3.1
+RUN pip install jsonformatter==0.3.2 gql[requests]==3.3.0
```

Comment:
correct, we haven't added gql in Dockerfile and it's not available in yum repos.